src/gui/hidabletabwidget.h

```diff
@@ -43,6 +43,16 @@ class HidableTabWidget : public QTabWidget
     }
 
 protected:
+#ifdef Q_OS_MAC
+    void paintEvent(QPaintEvent* event)
```

Comment:
Oh, you actually do use override and this was not intentional… Right, fixed for the rest as well.